db/migrate/20220902115804_create_response_items.rb

```diff
@@ -0,0 +1,13 @@
+class CreateResponseItems < ActiveRecord::Migration[7.0]
+  def change
+    create_table :response_items do |t|
+      t.belongs_to :query, index: true, foreign_key: true
+      t.integer :source, null: false
```

Comment:
thinking ahead, these should probably be indexed too, as we'll want to filter them in/out from the result set in a performant way, when the user is browsing and refining the result set.